webpack/JobWizard/steps/AdvancedFields/__tests__/AdvancedFields.test.js

```diff
@@ -297,4 +307,30 @@ describe('AdvancedFields', () => {
       }).value
     ).toBe('Run Default val');
   });
+
+  it('search resources action', async () => {
+    store.clearActions();
+    jest.useFakeTimers();
+
+    render(
+      <Provider store={store}>
+        <JobWizard />
+      </Provider>
+    );
+    await act(async () => {
+      fireEvent.click(screen.getByText(WIZARD_TITLES.advanced));
+    });
+    const resourceSelectField = screen.getByLabelText(
+      'adv resource select text input'
+    );
+
+    await act(async () => {
+      await fireEvent.change(resourceSelectField, {
+        target: { value: 'some search' },
+      });
+
+      jest.runAllTimers();
+    });
+    expect(store.getActions()).toMatchSnapshot('resource search');
```

Comment:
instead of taking a snapshot, can you test where the UI changes?

Reply:
I want to see that the correct action is fired as the ui changes are only happening from the api response.

Reply:
the approach of react testing library is to detect the UI changes rather than snapshots or implementation details,
IMHO we can mock the final API call and get results based on our search,
then we can expect to have the results in the DOM.

Reply:
I'm going to split the resource search into having a wrapper, and then test the wrapper it self with a mock responses, is it ok that we leave the test fixes for the next pr? (The host select)

Reply:
Sure, np 👍🏼